cmd/mavenExecuteStaticCodeChecks.go

```diff
@@ -78,15 +81,18 @@ func getSpotBugsMavenParameters(config *mavenExecuteStaticCodeChecksOptions) *ma
 
 func getPmdMavenParameters(config *mavenExecuteStaticCodeChecksOptions) *maven.ExecuteOptions {
 	var defines []string
-	if config.PmdExcludes != nil {
-		defines = append(defines, "-Dpmd.excludes="+strings.Join(config.PmdExcludes, ","))
+	if config.PmdMaxAllowedViolations != 0 {
+		defines = append(defines, "-Dpmd.maxAllowedViolations="+strconv.Itoa(config.PmdMaxAllowedViolations))
 	}
-	if config.PmdRuleSets != nil {
-		defines = append(defines, "-Dpmd.rulesets="+strings.Join(config.PmdRuleSets, ","))
+	if config.PmdFailurePriority >= 1 && config.PmdFailurePriority <= 5 {
+		defines = append(defines, "-Dpmd.failurePriority="+strconv.Itoa(config.PmdFailurePriority))
+	} else {
+		log.Entry().Warningf("Pmd failure priority must be a value between 1 and 5. %v was configured. Defaulting to 5.", config.PmdFailurePriority)
```

Comment:
Quick question: 5 is the default of the plugin itself and doesn't need to be configured?

Reply:
yes :)